### PR TITLE
#149304065 Tour guide

### PR DESCRIPTION
**Pull request by vynessa**
#### What does this PR do?
This enables users to be prompted with a guide on how to navigate through the website.

#### Description of Task to be completed?
* Make use of intro.js module for the tour guide
* Install tether-shepherd for position of elements correctly before the next
* Create the tour guide in tour.js
* Add styling to the tour page
* Import the react TourButtonComponent for start up guide

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
screen shot 2017-08-18 at 10 39 40 am
screen shot 2017-08-18 at 10 39 25 am

### CODE REVIEW

File: imports/plugins/core/ui-navbar/client/components/navbar/navbar.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
 Template.CoreNavigationBar.onCreated(function () {
   this.state = new ReactiveDict();
 });
 
+Template.CoreNavigationBar.onRendered(function () {
+  currentRoute = Router.getRouteName();
+  this.autorun(() => {
+    if (Accounts.findOne(Meteor.userId())) {
+      if (!Accounts.findOne(Meteor.userId()).takenTour && Accounts.findOne(Meteor.userId()).emails[0]) {
```
Comment (mel-michael) [inline imports/plugins/core/ui-navbar/client/components/navbar/navbar.js]: This line of code is difficult to read. Break the common parts into a variable. Essentially, don't repeat your self.

File: imports/plugins/included/tour/client/tour.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
+import introJs from "intro.js";
```
Comment (mel-michael) [inline imports/plugins/included/tour/client/tour.js]: Shouldn't this line be `import { IntroJS } from "intro.js"`?

File: imports/plugins/included/tour/client/tour.js [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+import { Reaction } from "/client/api";
+import { Meteor } from "meteor/meteor";
+import { Accounts } from "/lib/collections";
+
+const tour = introJs.introJs();
```
Comment (mel-michael) [inline imports/plugins/included/tour/client/tour.js]: if you use object destruction for line 1, you won't need to do this again. Your new code will be just `tour = IntroJS()`

Comment (vynessa) [reply imports/plugins/included/tour/client/tour.js]: True. Will update this now.

File: imports/plugins/core/ui-navbar/client/components/navbar/navbar.js (hunk 1 above)
Comment (vynessa) [reply imports/plugins/core/ui-navbar/client/components/navbar/navbar.js]: Noted.

**Pull request merged**